Formula/grokj2k.rb

```diff
@@ -77,23 +73,30 @@ def install
               "\\1 #{Formula["little-cms2"].opt_include}"
     end
 
-    # Workaround to fix build when using Homebrew little-cms2 headers with C++17.
-    # lcms2.h:1279:44: error: ISO C++17 does not allow 'register' storage class specifier
-    # See https://github.com/GrokImageCompression/grok/issues/241
-    ENV.append "CXXFLAGS", "-DCMS_NO_REGISTER_KEYWORD=1"
-
     perl = DevelopmentTools.locate("perl")
+    perl_archlib = Utils.safe_popen_read(perl.to_s, "-MConfig", "-e", "print $Config{archlib}")
+    args = %W[
+      -DGRK_BUILD_DOC=ON
+      -DGRK_BUILD_LIBPNG=OFF
+      -DGRK_BUILD_LIBTIFF=OFF
+      -DPERL_EXECUTABLE=#{perl}
+      -DCMAKE_INSTALL_BINDIR=libexec/bin
```

Comment:
```suggestion
      -DCMAKE_INSTALL_BINDIR=libexec
```
Do we put anything else in `libexec`? If not, then just install the binaries directly into `libexec` instead of adding an intermediate subdirectory.

Reply:
I might revert to installing into `bin` and move binary to `libexec` to simplify installation. There didn't seem to be any breakages with previous logic and using `libexec` instead of `libexec/bin` at least has a similar relative path.

Reply:
Yes, `libexec` is usually better to preserve relative paths.